tests/egl/gles2_help.c

```diff
@@ -23,6 +23,8 @@
 #include "config.h"
 #endif
 
+#if __ENABLE_TESTS_GLES__
+
```

Comment:
This check should be moved out from this file in the same way it was removed from the libyami patches.